frontend/src/App.tsx

```diff
@@ -58,6 +84,280 @@ interface IRouteGroup {
+    const [aboutDDIsOpen, aboutDDSetOpen] = useState<boolean>(false)
+
+    function DocsButton() {
+        return (
+            <ApplicationLauncherItem href="https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/">
+                Documentation
+            </ApplicationLauncherItem>
+        )
+    }
+    function AboutButton() {
+        return (
+            <ApplicationLauncherItem component="button" onClick={() => props.aboutClick()}>
+                About
+            </ApplicationLauncherItem>
+        )
+    }
+
+    return (
+        <ApplicationLauncher
+            aria-label="about-menu"
+            data-test="about-dropdown"
+            className="co-app-launcher co-about-menu"
+            onSelect={() => aboutDDSetOpen(false)}
+            onToggle={() => aboutDDSetOpen(!aboutDDIsOpen)}
+            isOpen={aboutDDIsOpen}
+            items={[<DocsButton key="docs" />, <AboutButton key="about_modal_button" />]}
+            data-quickstart-id="qs-masthead-helpmenu"
+            position="right"
+            toggleIcon={<QuestionCircleIcon style={{ color: '#EDEDED' }} />}
+        />
+    )
+}
+
+function UserDropdown() {
+    const [userIsOpen, userSetOpen] = useState<boolean>(false)
+
+    function configureClient() {
+        // Get the user token endpoint from the console backend to launch to the OCP Display Token page
+        const resp = configure()
+        resp.promise
+            .then((payload) => {
+                payload && payload.token_endpoint ? window.open(`${payload.token_endpoint}/request`, '_blank') : ''
+            })
+            .catch((error) => {
+                // eslint-disable-next-line no-console
+                console.error(error)
+            })
+    }
+
+    function logout() {
+        console.log('***** starting logout function')
+        // Get username so we know if user is kube:admin
+        let admin = false
+        const userResp = getUsername()
+        userResp.promise
+            .then((payload) => {
+                if (payload && payload.body && payload.body.username) {
+                    admin = payload.body.username === 'kube:admin'
+                }
+            })
+            .catch((error) => {
+                // eslint-disable-next-line no-console
+                console.error(error)
+            })
+        // Get user's oauth token endpoint
+        let oauthTokenEndpoint = ''
+        const configResp = configure()
+        configResp.promise
+            .then((payload) => {
+                payload && payload.token_endpoint ? (oauthTokenEndpoint = payload.token_endpoint) : ''
+            })
+            .catch((error) => {
+                // eslint-disable-next-line no-console
+                console.error(error)
+            })
+        const logoutUrl = getBackendUrl() + '/logout'
+        console.log(location.href)
+        console.log(location.origin)
```

Comment:
Some more console.logs to clean up. There are a bunch of others as well.

Reply:
Yes, was waiting to see if logout for `kube:admin` worked using those for debugging.  I will pull those out.

Reply:
console.log statements have been removed